stack/stack_verify.cpp

```diff
@@ -8,18 +8,18 @@ void stack_dump(my_stack *stk) {
     printf("hash for struct: %ld\n", stk->hash.hash_str);
     printf("hash for data: %ld\n", stk->hash.hash_data);
     printf("adress of 1st elem of stk: %p\n", stk->data);
-    printf("size of stk: %d\n", stk->size - 1);
-    printf("capasity of stk: %d\n", stk->capacity);
+    printf("size of stk: %ld\n", stk->size - 1);
+    printf("capasity of stk: %ld\n", stk->capacity);
```

Comment:
`%ld` is for decimal. there you have unsigned long long value. it is `%llu`, or maybe something else. I don't remember